### ISSUE TITLE
[FEATURE] build kimai version 2.0.14 

### ISSUE THREAD

**Issue by bykof** [tobybatch/kimai2#501]
**Is your feature request related to a problem? Please describe.**
I want to test kimai version 2.0.14 

**Describe the solution you'd like**
Rebuild the docker images and push the version 2.0.14 :)

**Additional context**
Thank you a lot for this repo!

**Comment by tobybatch** [comment]
2.x images are currently not supported. You can build your own, it shouldn't be too hard, something like:

```
docker build \
    -t my-pre-release-kimai:2.0.14 \
    --target=prod \
    --build-arg KIMAI=2.0.14 \
    --build-arg BASE=fpm \
    --build-arg TZ=Europe/London \
    .
```

Adjust the values as desired.

I'll look at the auto build but I don't want to impact the people using the `:latest` tag by mistake.

**Comment by bykof** [comment]
Alright thanks for the note!

**Comment by tobybatch** [comment]
@bykof Checkout #502 I'm building a custom image for this bug.

Do you want apache or fpm, prod or dev?

**Comment by bykof** [comment]
fpm prod :) thanks!

**Comment by tobybatch** [comment]
Closed in favout of #502
